Error in go.mod and go.sum

### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

When I try to install cerbos, I countered the following error:
```
joanne@joanne-t14p:~/cerbos$ go install ./cmd/cerbos
verifying github.com/jdxcode/netrc@v1.0.0/go.mod: checksum mismatch
        downloaded: h1:Gh9eFQJnoTNIRHXl2j5bJXA1u84hQWJWgGh569zF3v8=
        /home/joanne/cerbos/go.sum:     h1:Zi/ZFkEqFHTm7qkjyNJjaWH4LQA9LQhGJyF0lTYGpxw=
```

I copy the right checksum into go.mod, then it happens as:
```
cmd/cerbos/main.go:9:2: github.com/jdxcode/netrc@v1.0.0: parsing go.mod:
        module declares its path as: github.com/jdx/go-netrc
                but was required as: github.com/jdxcode/netrc
api/genpb/cerbos/policy/v1/policy.pb.go:13:2: github.com/jdxcode/netrc@v1.0.0: parsing go.mod:
        module declares its path as: github.com/jdx/go-netrc
                but was required as: github.com/jdxcode/netrc
```

So I change `jdxcode/netrc` into `jdx/go-netrc` in both go.mod and go.sum, and it works.

### Expected Behavior

_No response_

### Steps To Reproduce

_No response_

### Environment

```markdown
- OS:
- Cerbos version:
- Docker version:
- Go version:
```

### Anything else?

_No response_

--- a/go.mod
+++ b/go.mod
@@ -18,7 +18,7 @@ require (
 	github.com/aws/smithy-go v1.22.3
 	github.com/bluele/gcache v0.0.2
 	github.com/cenkalti/backoff/v5 v5.0.2
-	github.com/cerbos/cerbos-sdk-go v0.3.1
+	github.com/cerbos/cerbos-sdk-go v0.3.2
 	github.com/cerbos/cerbos/api/genpb v0.43.1-0.20250603061909-12f6de6ab927
 	github.com/cerbos/cloud-api v0.1.46
 	github.com/cespare/xxhash/v2 v2.3.0
@@ -216,7 +216,7 @@ require (
 	github.com/jackc/pgx/v4 v4.18.3 // indirect
 	github.com/jackc/puddle/v2 v2.2.2 // indirect
 	github.com/jbenet/go-context v0.0.0-20150711004518-d14ea06fba99 // indirect
-	github.com/jdxcode/netrc v1.0.0 // indirect
+	github.com/jdx/go-netrc v1.0.0 // indirect
 	github.com/jmespath/go-jmespath v0.4.0 // indirect
 	github.com/kevinburke/ssh_config v1.2.0 // indirect
 	github.com/klauspost/compress v1.18.0 // indirect

